.github/workflows/lint.yml

```
@@ -12,15 +12,56 @@ env:
+      AWS_CONSOLE_ENDPOINT: http://localhost:9001
+      AWS_TEST_BUCKET: testbucket
+      CI: true
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
+    - name: Install latest PostgreSQL client
+      run: |
+        sudo apt-get update
+        sudo apt-get install --yes postgresql-client
     - name: Install Rust
       uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)